Exercise_D and E/index.css

```diff
@@ -0,0 +1,87 @@
+nav {
+    height: 80px;
+    background: white;
+    padding: 0px 20px;
+    display: flex;
+    align-items: center;    
+}
+
+nav img{
+    height: 70px;
+}
+
+header {
+    background-image:    linear-gradient(
+        rgba(0, 0, 0, 0.6), 
+        rgba(0, 0, 0, 0.6)
+      ), url("./assets/images/flowershop.jpg");
+    height: 800px;
+    background-position: center; /* Center the image*/
+    background-repeat: no-repeat; /* Do not repeat the image*/
+    background-size: cover; /* Resize the background image to cover the entire container */
+    align-content: center;
+    justify-content: center;
+    display: flex;
+    color: white;
+    font-weight: bold;
+    flex-flow: row wrap;
+}
+
+.break {
+    flex-basis: 100%;
+    height: 20px;
+}
+
+.button {
+    background-color: greenyellow;
+    height:60px;
+    width: fit-content;
+    text-align: center;
+    padding: 12px 15px 10px 15px;
+    border-radius: 30px;
+}
+
+.button a{
+    text-decoration: none;
+    color: white;
```

Comment:
You may be trying to remove the css when you hover over the element?
If so, 
 try something like .button a: hover {
 text-direction: none
 }